src/components/dropdown/dropdown.ts

```diff
@@ -175,7 +167,6 @@ export default class IgcDropDownComponent
   public override firstUpdated() {
     if (this.targetNodes.length) {
       this.target = this.targetNodes[0];
-      // this.target.setAttribute('aria-owns', 'igcScrollContainer');
```

Comment:
Maybe we should keep this line, depending on what we do with ARIA down the road, not sure how reliable this would be